generate_outputs.py

```diff
@@ -178,63 +182,76 @@
     log.info("Auto Coding Demogs...")
     data = AutoCodeDemogs.auto_code_demogs(user, data, phone_number_uuid_table, coded_dir_path)
 
-    log.info("Applying Manual Codes from Coda...")
-    data = ApplyManualCodes.apply_manual_codes(user, data, prev_coded_dir_path)
+    if pipeline_configuration.analysis_files_mode:
```

Comment:
We need to export some traced data even if we don't generate analysis files

Reply:
also this if/else is very long - is there anything that can be done to break it up?

Reply:
Have updates this to generate autocoding Traceddata file if generate_analysis_files == 'false'